dpnp/dpnp_iface_statistics.py

```diff
@@ -76,6 +76,32 @@
 ]
 
 
+def _check_limitations(initial=None, where=True):
```

Comment:
@antonwolfy  I added it here because `pylint` was complaining about duplicated code in `dpnp_iface_statistics.py` and `dpnp_iface.nanfunctions.py`. (both files should be changed in  a same commit for `pylint` to compare them, if we modify one file at at a time for each commit, `pylint` only checks the modified file and does not compared it with other ones).

Anyway, I defined `_check_limitations` here and import it in `dpnp_iface.nanfunctions.py`. We also have a similar function in `dpnp_iface.arraycreation.py` but with different keywords. Should we combine them and put them for instance in `dpnp_iface.py` to be accessible by all files by calling `dpnp.check_limitations` or you are good with the current implementation?